cli/cppcheckexecutor.cpp

```diff
@@ -924,8 +924,7 @@ int CppCheckExecutor::check_internal(CppCheck& cppcheck, int /*argc*/, const cha
             totalfilesize += i->second;
         }
 
-        std::size_t processedsize = 0;
-        unsigned int c = 0;
+        std::size_t processedsize = 0, c = 0;
```

Comment:
varför `std::size_t c` .. det är ingen data-storlek. det är en räknare.

Reply:
Because it is passed to a function that takes size_t?

Reply:
I spotted it because MSVC warned about a potential overflow.

Reply:
imho I do not feel that c should be a std::size_t .. I'd rather make it int and see if we can make MSVC happy. There will not be more than `2^32` files to check.